scripts/run-e2e.ts

```diff
@@ -82,8 +83,6 @@ const configureYarn2 = async ({ cwd }: Options) => {
     // See https://github.com/yarnpkg/berry/pull/2078
     // As soon as a new version of Yarn is released remove next line
     `yarn set version from sources --branch 2078`,
-    // ⚠️ Need to set registry because Yarn 2 is not using the conf of Yarn 1
-    `yarn config set npmScopes --json '{ "storybook": { "npmRegistryServer": "http://localhost:6000/" } }'`,
     // Some required magic to be able to fetch deps from local registry
     `yarn config set unsafeHttpWhitelist --json '["localhost"]'`,
```

Comment:
If all the e2e tests ran using Yarn 2 you would add the following command here
```
yarn link "/path/to/storybook" --all
```
Then if the test doesn't support PnP yet run
```
yarn config set nodeLinker node-modules
```